src/components/CommentInput/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { StyleSheet, TextInput, View, ViewStyle } from 'react-native';
+
+import { useInput } from '../../utils/hook';
+
+interface Props {
+	style?: ViewStyle;
+	containerStyle?: ViewStyle;
+}
+
+export const CommentInput: FC = ({
+	style = {},
+	containerStyle = {},
```

Comment:
it's ok to let `style` and `containerStyle` empty (`undefined`), skip the empty Object fallback.

Reply:
I've fixed it, pls check again